content/docs/reference-react-dom.md

```diff
@@ -80,28 +80,28 @@ Remember to be mindful of user experience on slow connections. The JavaScript co
 ReactDOM.unmountComponentAtNode(container)
 ```
 
-Remove a mounted React component from the DOM and clean up its event handlers and state. If no component was mounted in the container, calling this function does nothing. Returns `true` if a component was unmounted and `false` if there was no component to unmount.
+Membuang sebuah komponen React terpasang dari DOM dan membersihkan *event handlers* dan *state* terkait. Jika tidak terdapat komponen yang terpasang di dalam *container*, memanggil *function* ini tidak melakukan apa-apa. Memulangkan `true` jika sebuah komponen dicabut dan `false` jika tidak ada komponen yang dicabut.
 
 * * *
 
 ### `findDOMNode()` {#finddomnode}
 
-> Note:
+> Catatan:
 >
-> `findDOMNode` is an escape hatch used to access the underlying DOM node. In most cases, use of this escape hatch is discouraged because it pierces the component abstraction. [It has been deprecated in `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
+> `findDOMNode` adalah sebuah cara yang digunakan untuk mengakses *DOM node* pokok. Pada umumnya, penggunaan cara ini tidak disarankan sebab menembus abstraksi komponen. [Sudah tidak berlaku dalam `StrictMode`.](/docs/strict-mode.html#warning-about-deprecated-finddomnode-usage)
 
 ```javascript
 ReactDOM.findDOMNode(component)
 ```
-If this component has been mounted into the DOM, this returns the corresponding native browser DOM element. This method is useful for reading values out of the DOM, such as form field values and performing DOM measurements. **In most cases, you can attach a ref to the DOM node and avoid using `findDOMNode` at all.**
+Jika komponen ini telah dipasang ke dalam DOM, memulangkan elemen DOM asli browser yang terkait. Metode ini berguna untuk membaca nilai keluar dari DOM, seperti nilai-nilai form field dan melakukan pengukuran DOM. **Pada umumnya, Anda dapat melampirkan sebuah referensi ke *DOM node* dan menghindari penggunaan `findDOMNode` sepenuhnya.**
```

Comment:
memulangkan -> mengembalikan
browser -> *browser*
seperti nilai-nilai form field -> @regalius ?
performing DOM measurements -> melakukan pengukuran DOM -> @regalius ?

Reply:
seperti nilai-nilai form field -> seperti nilai-nilai isian *form*


melakukan pengukuran DOM -> Have a little doubt about this one, the context of this sentence is that we can measure the size of the DOM element itself by using the returned DOM from `findDOMNode`

Reply:
```suggestion
Jika komponen ini telah terpasang ke dalam DOM, mengembalikan elemen DOM asli *browse*r yang terkait. Metode ini berguna untuk membaca nilai hasil dari DOM, seperti nilai-nilai isian form dan melakukan perhitungan DOM. **Pada umumnya, Anda dapat melampirkan sebuah referensi ke *DOM node* dan menghindari penggunaan `findDOMNode` sepenuhnya.**
```

Reply:
Hmm `perhitungan DOM` can also be interpreted as `counting the number of the DOMs` so I think it's better to keep it as `pengukuran` or `penghitungan ukuran` instead mas @anggras

Reply:
Sip :ok_man: